corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -245,14 +260,23 @@ def _looks_like_markdown(self, str):
     def _get_markdown_node(self, text_node_):
         return text_node_.find("./{f}value[@form='markdown']")
 
+    @staticmethod
+    def _get_default_value_nodes(text_node_):
+        for value_node in text_node_.findall("./{f}value"):
+            if 'form' not in value_node.attrib:
+                yield value_node
+            elif value_node.get('form') == 'default':
+                # migrate invalid values, http://manage.dimagi.com/default.asp?236239#BugEvent.1214824
+                value_node.attrib.pop('form')
+                yield value_node
```

Comment:
need an else statement to yield the value_node if `form` is something other than `default`

Reply:
Won't that change the logic that was there earlier?

Reply:
true